### ISSUE TITLE
Arbitrary File Write vulnerability

### ISSUE THREAD

**Issue by WilliamDASILVA** [vuejs/vue-cli#5267]
### Version
3.10.0

### Environment info
```
System:
    OS: Linux 4.9 Debian GNU/Linux 8 (jessie) 8 (jessie)
    CPU: (5) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
  Binaries:
    Node: 10.9.0 - /usr/local/bin/node
    Yarn: 1.9.2 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: Not Found
    Firefox: Not Found
  npmPackages:
    @storybook/vue: ^5.1.3 => 5.1.11 
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0 
    @vue/babel-plugin-transform-vue-jsx:  1.0.0 
    @vue/babel-preset-app:  3.10.0 
    @vue/babel-preset-jsx:  1.1.0 
    @vue/babel-sugar-functional-vue:  1.0.0 
    @vue/babel-sugar-inject-h:  1.0.0 
    @vue/babel-sugar-v-model:  1.0.0 
    @vue/babel-sugar-v-on:  1.1.0 
    @vue/cli: ^3.10.0 => 3.10.0 
    @vue/cli-overlay:  4.1.1 
    @vue/cli-plugin-babel: ^3.10.0 => 3.10.0 
    @vue/cli-plugin-eslint: ^3.10.0 => 3.10.0 
    @vue/cli-plugin-pwa: ^3.10.0 => 3.10.0 
    @vue/cli-plugin-router:  4.1.1 
    @vue/cli-plugin-unit-jest: ^4.0.4 => 4.0.4 
    @vue/cli-plugin-vuex:  4.1.1 
    @vue/cli-service: ^4.1.1 => 4.1.1 
    @vue/cli-shared-utils:  3.10.0 (4.1.1, 4.0.4)
    @vue/cli-ui:  3.10.0 
    @vue/cli-ui-addon-webpack:  3.10.0 
    @vue/cli-ui-addon-widgets:  3.10.0 
    @vue/component-compiler-utils:  3.1.0 
    @vue/eslint-config-standard: ^4.0.0 => 4.0.0 
    @vue/preload-webpack-plugin:  1.1.1 
    @vue/test-utils: ^1.0.0-beta.26 => 1.0.0-beta.29 
    @vue/web-component-wrapper:  1.2.0 
    babel-helper-vue-jsx-merge-props:  2.0.3 
    babel-plugin-transform-vue-jsx:  3.7.0 
    babel-preset-vue: ^2.0.2 => 2.0.2 
    bootstrap-vue: ^2.1.0 => 2.1.0 
    eslint-plugin-vue: ^5.0.0 => 5.2.2 (4.7.1)
    jest-serializer-vue: ^0.3.0 => 0.3.0 (2.0.2)
    portal-vue:  2.1.6 
    storybook-vue-router: ^1.0.3 => 1.0.3 
    typescript:  3.7.2 
    vue: ^2.6.10 => 2.6.10 
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
Run the "npm audit" command.

### What is expected?
No vulnerability warning.

### What is actually happening?
Vulnerability found in the package "decompress".
Since the maintainer of "decompress" and the parent dependency "download" are no longer active; the package maintainer of "download-git-repo" do not respond, the vulnerability cannot be solved.

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Arbitrary File Write                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ decompress                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @vue/cli [dev]                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @vue/cli > download-git-repo > download > decompress         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1217                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in 6097935 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

---
A solution would be to find an alternative to "download-git-repo"

**Comment by sodatea** [comment]
Note: this vulnerability is not of high risk in the context of Vue CLI. See https://github.com/vuejs/vue-cli/issues/5249#issuecomment-596167406

**Comment by surapuramakhil** [comment]
what would be the fix it would be? is it using the alternative of download-git-repo?  
or wait till download-git-repo to respond

**Comment by sodatea** [comment]
So, now that `decompress` has released 4.2.1 to address this vulnerability, I think this issue can be marked as resolved.
